lib/core/test/bench/db/Main.hs

```diff
@@ -607,12 +606,16 @@ instance NFData SqliteContext where
 testCp :: WalletBench
 testCp = snd $ initWallet block0 genesisParameters initDummyState
 
+{-# NOINLINE initDummyState #-}
 initDummyState :: SeqState 'Testnet ShelleyKey
 initDummyState =
     mkSeqState (xprv, mempty) defaultAddressPoolGap
   where
-    bytes = entropyToBytes <$> unsafePerformIO $ genEntropy @(EntropySize 15)
-    xprv = generateKeyFromSeed (Passphrase bytes, mempty) mempty
+    mnemonic = unsafePerformIO $
+        SomeMnemonic
+        . entropyToMnemonic @15
+        <$> genEntropy @(EntropySize 15)
```

Comment:
Minor style suggestion: 

This is a relatively weird choice of indentation. It doesn't seem to improve much readability. The composition is at the same level as the applicative, whereas it's really the composition that is applied to value inside the IO from `genEntropy`. So, what about grouping things as:

```hs
mnemonic = unsafePerformIO 
    $ SomeMnemonic . entropyToMnemonic @15 
   <$> genEntropy @(EntropySize 15)
```